packages/flutter/lib/src/semantics/semantics.dart

```diff
@@ -3084,6 +3123,12 @@ class SemanticsNode with DiagnosticableTreeMixin {
 
         thickness = math.max(thickness, node._thickness + node._elevation);
 
+        if (controlsVisibilityOfNodes == null) {
+          controlsVisibilityOfNodes = node._controlsVisibilityOfNodes;
```

Comment:
Can `node._controlsVisibilityOfNodes` be mutated? If so, should we create a defensive copy here?